models/article-models.js

```diff
@@ -1,19 +1,21 @@
 const db = require('../db/connection'); 
 
 const fetchArticleByArticleID = (id) => {
-   return db.query('SELECT * FROM articles WHERE article_id = $1;', [id])
+   return db.query('SELECT articles.title, articles.topic, articles.author, articles.created_at, articles.votes, comments.article_id, (SELECT COUNT(comment_id) :: INT) AS comment_count FROM articles INNER JOIN comments ON articles.article_id = comments.article_id WHERE articles.article_id = $1 GROUP BY articles.article_id, comments.article_id', [id])
```

Comment:
looks fine to me. TOP TIP you can write your string in back ticks which allow you add line breaks so the formatting is a little nicer